packages/react-tabster/src/hooks/useFocusIndicatorStyle.ts

```diff
@@ -0,0 +1,17 @@
+import { makeStyles } from '@fluentui/react-make-styles';
+import { KEYBOARD_NAV_SELECTOR } from '../symbols';
+
+const useStyles = makeStyles({
+  focus: theme => ({
+    outline: 'none',
+    [`:global(${KEYBOARD_NAV_SELECTOR}) :focus`]: {
+      outline: `solid 1px ${theme.alias.color.neutral.neutralForeground1}`,
+    },
+  }),
+});
+
+/**
+ * Returns className for focus indicator if user is using keyboard navigation
+ * otherwise returns an empty string
+ */
+export const useFocusIndicatorStyle = (): string => useStyles().focus;
```

Comment:
```suggestion
export const useFocusIndicatorClassname = (): string => useStyles().focus;
```

nit: As it returns a classname there is sense to rename it. Feel free to ignore

Reply:
I used style postfix since all converged components are using style postfix for their hooks involving CSS classnames